frontend/src/components/activePage/Instances/InstanceContent/ManagedInstanceHeading.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
+import { Typography } from 'antd';
+
+export interface IManagedInstanceHeadingProps {
+  tenantId: string;
+  tenantDisplayName: string;
+  displayName: string;
+}
+
+const ManagedInstanceHeading: FC<IManagedInstanceHeadingProps> = ({
```

Comment:
If you need a different component, create a new folder (I guess you referred tyo this in the PR description)